### PR TITLE
#2101 Extend Interpreter View popup menu

### PR DESCRIPTION
**Pull request by GlennPlou**
Change-Id: I327db2f281bd5b7a63780bc1778429f28dc24283
Signed-off-by: Glenn Plouhinec <glenn.plouhinec@obeo.fr>

### CODE REVIEW

**Comment by GlennPlou** [comment]
The new tools of this PR extend the Interpreter View and will be operational in the next release of Sirius 6.5.0 which will integrate Acceleo 3.8.

File: core/plugins/org.polarsys.capella.core.sirius.ui/src/org/polarsys/capella/core/sirius/ui/handlers/CopyTextHandler.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  /**
+   * @return the current selection represented as text
+   */
+  private String getSelectionAsText() {
+    StringBuilder builder = new StringBuilder();
+    String result = null;
+    IStructuredSelection selection = getSelection();
+    for (Object o : selection.toArray()) {
+      if (o instanceof ModelElement) {
+        builder.append(((ModelElement) o).getFullLabel());
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.sirius.ui/src/org/polarsys/capella/core/sirius/ui/handlers/CopyTextHandler.java:52]: getLabel

**Review by pdulth — CHANGES_REQUESTED** [review]

File: core/plugins/org.polarsys.capella.core.sirius.ui/plugin.properties [hunk 2]
```diff
@@ -10,14 +10,17 @@
...
-providerName = Eclipse.org
+capella.fragment.editor.name=Capella Fragment Model Editor
+exportMenuLabel=Export
+exportToCsvLabel=CSV
+providerName = Eclipse.org
 pluginName = Capella Sirius UI Plug-in
 
 Capella_Rename_Participant_name=Capella Rename Participant
 
+command.copyAsText=Copy
```
[the remark this answers is not in the document]
Comment (GlennPlou) [reply core/plugins/org.polarsys.capella.core.sirius.ui/plugin.properties]: Done.

File: core/plugins/org.polarsys.capella.core.sirius.ui/plugin.xml [hunk 3]
```diff
@@ -254,5 +273,13 @@
             icon="icons/viewpoint.gif">
       </image>
    </extension>
+   <extension
+         point="org.eclipse.ui.bindings">
```
[the remark this answers is not in the document]
Comment (GlennPlou) [reply core/plugins/org.polarsys.capella.core.sirius.ui/plugin.xml]: Done.

File: core/plugins/org.polarsys.capella.core.sirius.ui/src/org/polarsys/capella/core/sirius/ui/handlers/CopyTextHandler.java (hunk 1 above)
Comment (GlennPlou) [reply core/plugins/org.polarsys.capella.core.sirius.ui/src/org/polarsys/capella/core/sirius/ui/handlers/CopyTextHandler.java]: Done.

**Comment by GlennPlou** [comment]
The commit has been amended. Recommendations have been taken into account.

**Review by pdulth — APPROVED** [review]

**Pull request merged**